_config.yml

```diff
@@ -18,12 +18,12 @@
 # Feel free to change these! For more info, refer to:
 # ~> https://jekyllrb.com/docs/configuration/options/
 
-title:            'Alberta COVID-19 Community Archive'
-description:      'A Visual Record During the COVID-19 Pandemic'
+title:            '/img/UA-LIBR-COLOUR.png Alberta COVID-19 Community Archive'
+description:      'A Visual Record of Alberta During the COVID-19 Pandemic'
 url:              'https://pbinkley.github.io'
 baseurl:          '/wax-community-archive'
-copyright:        'Example copyright org, 2019'
-logo:             '/assets/logo.png'
+copyright:        ''/img/cc-by-nc-sa.png This work is subject to a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International license
```

Comment:
Single quotes need to surround the string; and the img path again

Reply:
The new commit has the path and then the text, quotated separately. YAML requires just one string here, i.e. the text.